src/topology/bornology/order.lean

```diff
@@ -0,0 +1,190 @@
+/-
+Copyright (c) 2022 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+-/
+import topology.bornology.basic
+
+/-!
+# Complete lattice structure on bornologies
+
+In this file, we put a `complete_lattice` structure on `bornology α`. Following mathlib
+conventions, we have `b₁ ≤ b₂` if and only if `id : (α, b₁) → (α, b₂)` is a `locally_bounded_map`,
+i.e larger bornologies have more bounded sets.
+
+We also define the bornology generated by a family of sets `S` as the smallest bornology for which
+all elements of `S` are bounded.
+-/
+
+
+open set filter function
+
+variables {α β γ : Type*} {b₁ b₂ : bornology α} {s t u v : set α}
+
+namespace bornology
+
+lemma cobounded_injective : injective (@cobounded α) :=
+λ b₁ b₂ h, by ext; rw h
+
+instance : partial_order (bornology α) := partial_order.lift
+  (_ : bornology α → order_dual (filter α)) cobounded_injective
```

Comment:
I am growing increasingly dubious that filters have the correct order in mathlib.

Reply:
But you are wrong. For all we do in topology, filters are generalizations of sets. The inclusion `filter.principal` of sets into filters has to be order preserving. 

For instance the definition of "a sequence `u : ℕ →X` converges to `x` is `map u at_top ≤ 𝓝 x` which reads "the direct image under `u` of 'the set of very large numbers in ℕ' is contained in 'the set of points that are close to x'". Putting the wrong order as you suggests simply cuts off all intuition from filters. 

The fact that `filter X` can be injected into `set (set X)` is an implementation detail. The important map is `filter.principal : set X → filter X`, not `filter.sets : filter X → set (set X)`.

What the line above suggests is that Anatole may be putting the wrong order relation on `bornology`.